.github/workflows/integration_tests_with_db.yaml

```diff
@@ -11,6 +11,18 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
maybe we could use a slimmer image like `alpine`?

Reply:
Not a docker image, only GitHub provided images supported and they add all the garbage

Reply:
right, my bad :grimacing:. sry cache was still occupied by the previous context :sweat_smile: